src/reducers/walletConnectReducer.js

```diff
@@ -45,11 +45,72 @@ export type WalletConnectReducerState = {
   waitingRequest: ?string,
 };
 
-export type WalletConnectReducerAction = {
-  type: string,
-  payload: any,
+type SessionApproved = {
+  type: 'WALLETCONNECT_SESSION_APPROVED',
+  payload: {
+    pending: Array<WalletConnect>,
```

Comment:
```suggestion
    pending: WalletConnect[],
```
to keep the consistency with other project types

Reply:
Oops, you're right. Updated.